index.html

```diff
@@ -19,8 +19,166 @@ <h1>Sebastián<br>Muñoz</h1>
     <section class="intro">
         <p>Hello! I am a software developer! I can help you build a product, feature or website.      Taka a look of my works. <br> If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </section>
-    <div class="collab">
+    <div class="btn">
         <button type="button">Start collaboration</button>
     </div>
+    <div class="geometry">
+        <img src="assets/img/image_geometry_2.png" alt="">
+    </div>
+    <section class="projects-title">
+        <h1>Projects</h1>
```

Comment:
- [OPTIONAL ] For semantic and SEO reasons the `h1` tag should be replaced with an `hx` tag since it was already at the headline. A page only contains one `h1` tag. Ensure you also correct this in the About section.